Quaver.Shared/Screens/Select/SelectScreen.cs

```diff
@@ -351,6 +363,32 @@ private static void HandleKeyPressTab()
                 ConfigManager.LeaderboardSection.Value = LeaderboardType.Local;
         }
 
+        /// <summary>
+        ///     Handles when the user presses the del key
+        /// </summary>
+        private void HandleKeyPressDel()
+        {
+            if (!KeyboardManager.IsUniqueKeyPress(Keys.Delete) || DialogManager.Dialogs.Count > 0)
+                return;
+
+            // osu! loaded map check to avoid errors due to the map loaded being virtual from osu.
+            if (MapManager.Selected.Value.Game == MapGame.Osu)
+            {
+                // Display error message
+                NotificationManager.Show(NotificationLevel.Error,$"This map cannot be deleted due to it being loaded directly from osu!");
```

Comment:
Avoid mentioning the names of other games here.

Reply:
And while you're at it add the missing space :jerry: